src/data.rs

```diff
@@ -367,6 +368,17 @@ impl_hash! { for FixedArray }
 impl_partial_eq! { Data for FixedArray use identity }
 impl_partial_eq! { FixedArray for FixedArray use identity }
 
+#[repr(C)]
+#[derive(Debug)]
+pub struct ModuleRequest(Opaque);
+
+impl_deref! { Data for ModuleRequest }
+impl_from! { Data for ModuleRequest }
```

Comment:
Not sure if this is correct... In v8 `ModuleRequest` has following method definition: `V8_INLINE static ModuleRequest* Cast(Data* data);`. Is that correct?

Reply:
Yes, it's correct -`ModuleRequest` is-a `Data`.